src/test/regress/sql/gporca.sql

```diff
@@ -3706,6 +3706,33 @@ WITH e AS (
 
 DROP TABLE d, r;
 
+-- Test that optimizer_enable_dml_triggers GUC is defunct 
+CREATE TABLE d (с1 int, с2 int) DISTRIBUTED BY (с1);
+-- The next query should be processed by Orca
+INSERT INTO d VALUES (1, 1);
+
+CREATE OR REPLACE FUNCTION trig_proc() RETURNS TRIGGER
+AS $$
+BEGIN
+  RAISE NOTICE 'Values: (%, %)', OLD.с1, OLD.с2;
+  RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER d_upd AFTER UPDATE ON d
+FOR EACH ROW EXECUTE PROCEDURE trig_proc();
+
+-- GUC should be disabled by default and cannot be enabled
+RESET optimizer_enable_dml_triggers;
```

Comment:
We assume that all GUSs has default values before tests, so this RESET is redundant